src/DynamoApplications/StartupUtils.cs

```diff
@@ -491,5 +491,37 @@ private static List<Exception> GetVersionMismatchedReferencesInAppDomain(Assembl
             }
             return output;
         }
+
+        /// <summary>
+        /// Attempts to import assemblies as node libraries from a given set of file paths.
+        /// </summary>
+        /// <param name="model">The Dynamo model</param>
+        /// <param name="paths">The list of paths from which to attempt importing </param>
+        public static void ImportAssemblies(DynamoModel model, IEnumerable<string> paths)
+        {
+            ArgumentNullException.ThrowIfNull(model, nameof(model));
+            ArgumentNullException.ThrowIfNull(paths, nameof(paths));
+
+            foreach(var path in paths)
+            {
+                try
+                {
+                    var filePath = new System.IO.FileInfo(path);
+                    if (!filePath.Exists)
+                    {
+                        model.Logger?.Log($"Could not find requested import library at path{path}");
+                    }
+                    else
+                    {
+                        var assembly = Assembly.LoadFile(path);
+                        model.LoadNodeLibrary(assembly, true);
```

Comment:
or maybe I should add a new method directly in the model?

Reply:
There's a public method in `ExtensionLibraryLoader` that essentially redirects the call to `DynamoModel.LoadNodeLibrary`.